pkgs/development/compilers/hip/default.nix

```
@@ -172,6 +169,14 @@ stdenv.mkDerivation (finalAttrs: {
     #!/usr/bin/env nix-shell
     #!nix-shell -i bash -p curl jq common-updater-scripts nix-prefetch-github
     version="$(curl ''${GITHUB_TOKEN:+"-u \":$GITHUB_TOKEN\""} -sL "https://api.github.com/repos/ROCm-Developer-Tools/HIP/tags" | jq '.[].name | split("-") | .[1] | select( . != null )' --raw-output | sort -n | tail -1)"
+
+    IFS='.'
+    read -a version_arr <<< "$version"
```

My bash is a bit rusty but doesn't

```suggestion
    IFS='.' read -a version_arr <<< "$version"
```

work?

---

Mine is too, AFAIK we won't have to ``unset IFS`` if we do this.